readme.md

```diff
@@ -33,8 +33,16 @@ packageJson('@company/package', 'latest').then(json => {
 ### packageJson(name, [version])
 
 You can optionally specify a version (e.g. `1.0.0`) or `latest`.  
-If you don't specify a version you'll get the [main entry](http://registry.npmjs.org/pageres/) containing all versions.
+If you don't specify a version you'll get the [main entry](https://registry.npmjs.org/pageres/) containing all versions.
 
+If the package is not scoped, the version can be in the form of:
+
+* `1` - get the latest `1.x.x`
+* `1.2` - get the latest `1.2.x`
+* `^1.2.3` - get the latest `1.x.x` but at least `1.2.3`
+* `~1.2.3` - get the latest `1.2.x` but at least `1.2.3`
+
+(Currently only full version numbers are supported for scoped packages.)
```

Comment:
Why they are supported, I have no idea, but apparently they are since you are already using them. What I meant by that was that scoped packages work after the PR just as they used to work before the PR, i.e. they still need a full version number. In other words, the PR doesn't change the behavior of the module when working with scoped packages.